test/adapter/json_api/has_many_links_test.rb

```diff
@@ -0,0 +1,250 @@
+require 'test_helper'
+
+module ActionController
+  module Serialization
+    class JsonApiHasManyLinksTest < ActionController::TestCase
+      LinkTagSerializer = Class.new(ActiveModel::Serializer) do
+        has_many :posts, links: { related: proc { |object, name| "http://test.host/tags/#{object.id}/#{name}" } }, data: false
+      end
```

Comment:
plz just `class LinkTagSerializesr < AM::S`. we're moving away from that weird Class.new(AM::S)